atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLockTables.java

```diff
@@ -77,6 +111,7 @@ private void createTableInternal(Cassandra.Client client, TableReference tableRe
         CassandraKeyValueServices.waitForSchemaVersions(
                 config,
                 client,
-                tableRef.getQualifiedName());
+                tableRef.getQualifiedName(),
+                true);
```

Comment:
To avoid the proliferation of boolean arguments, I'd prefer for there to be two methods, `waitForSchemaVersionsFromAllNodes` and `waitForSchemaVersionsFromQuorum` - this refactor is beyond the scope of the PR, however. Would be happy to see it done after merging!